Sources/Sentry/SentryThreadInspector.m

```diff
@@ -119,10 +119,19 @@ - (SentryStacktrace *)stacktraceForCurrentThreadAsyncUnsafe
         thread_act_array_t suspendedThreads = NULL;
         mach_msg_type_number_t numSuspendedThreads = 0;
 
-        sentrycrashmc_suspendEnvironment(&suspendedThreads, &numSuspendedThreads);
+        // We add a limit of 70 threads because in every test up to a 100 seems fine.
+        // We are giving it an extra safety margin.
```

Comment:
```suggestion
        // SentryThreadInspector is crashing when there is too many threads.
        // We add a limit of 70 threads because in test with up to 100 threads it seems fine.
        // We are giving it an extra safety margin.
```